### PR TITLE
Adding an assets panel to the lookbook demo to show assets for the component

### PR DESCRIPTION
**Pull request by jonrohan**
### Description

image

This PR adds an "Assets" [custom panel](https://lookbook.build/guide/extend/panels/) to the demo app that was mostly copied from the lookbook example app. 

[Preview URL](http://primer-view-components-preview-1578.eastus.azurecontainer.io/view-components/lookbook/inspect/primer/beta/button/playground)

### Integration

> Does this change require any updates to code in production?

### Merge checklist

- [x] Added/updated tests
- [x] Added/updated documentation
- [x] Added/updated previews

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -17,6 +17,6 @@ module.exports = {
         'focus-visible-pseudo-class': false
       }
     }),
-    require('cssnano'),
+    process.env.CI === 'true' ? require('cssnano') : null
```
Comment (jonrohan) [inline postcss.config.js:20]: I'm trying to get the CSS to not be compressed output for the new panel, my thinking here is to set to `null` in the proper circumstances.

**Review by langermank — APPROVED** [review]
Love this!

File: postcss.config.js (hunk 1 above)
Comment (camertron) [reply postcss.config.js:20]: What if we just read in the un-compiled .pcss file instead of the compiled .css file?

**Review by camertron — APPROVED** [review]
Super cool!

File: postcss.config.js (hunk 1 above)
Comment (jonrohan) [reply postcss.config.js:20]: Was thinking if we're sharing the compiled CSS it would be easier to copy/paste to a prototyping tool like Codepen.

**Pull request merged**